### DISCUSSION TITLE
Support celestial coordinates

### DISCUSSION THREAD

**Discussion by KellyStathis** [datacite/datacite-suggestions#37] [Metadata Schema Suggestions]
### What is the problem that your suggestion solves?

Most of our data refer to a specific area or object in the sky. Being able to provide that information for our data sets would be helpful for the astronomical community.

### What solution might meet your needs?

Allow location to be provided in celestial coordinates, rather than just terrestrial. I would suggest Right Ascension and Declination in the ICRS frame

### What alternatives have you tried or considered?

We just use the geolocation items

### Is there anything else you would like to share?

*Suggestion was previously submitted by a user. For DataCite staff reference: https://docs.google.com/spreadsheets/d/1ItM7bYocla83TCYmHI9oZ9Kgt7X0Lznfv6Zls7Gmi88/edit?resourcekey=&gid=1473745305#gid=1473745305*

### What group(s) would benefit from your suggestion?

- [ ] DataCite members registering DOIs and metadata and administering accounts
- [ ] Developers building integrations with DataCite services
- [ ] Harvesters who index DataCite DOIs and metadata
- [ ] Individuals or institutions analyzing DataCite DOIs and metadata
- [ ] DataCite staff
- [ ] Other

**Comment by rebeccaringuette** [comment]
We have a similar problem in Heliophysics. We indicate our locations in several different coordinate systems, some having 2 dimension and some having three. Suggest generalizing the coordinates to be coordinate 1 (e.g. X or latitude), coordinate 2 (e.g. Y or longitude), and coordinate 3 (e.g. Z, radius, or altitude). In addition, a field to indicate the coordinate system should be added (e.g. coordinateSystem). The indication of a term from a controlled vocabulary should be supported in that field (e.g. copy the structure from Subject). This approach should solve the problem for a larger range of sciences - astrophysics, heliophysics, planetary science, and so on.

**Comment by NRayya** [comment]
We also need three dimensional coordinates for environmental samples, it would be great if we get the altitude added.
